### ISSUE TITLE
Color code books in Kourend Library plugin

### ISSUE THREAD

**Issue by Bownerator** [runelite/runelite#12557]
When doing Kourend Library as a training method, it's vital to collect books as you go past them, and to do this most effectively you need to know where all the missing books are. Making the book titles change color based on their location (Northwest, Southwest, Northeast or Center) would make it much easier to quickly plan a route to collect missing books. The books should return to the original color if you have a copy in your inventory.

**Comment by Nightfirecat** [comment]
I don't think this would be the best way to address this need. I think it would be better to add some kind of icon or other clearer indicator of the location which can be more generally understood than a color reference system.
